helios-services/src/main/java/com/spotify/helios/agent/TaskConfig.java

```diff
@@ -112,7 +112,7 @@ public ContainerConfig containerConfig(final ImageInfo imageInfo,
     builder.hostname(job.getHostname());
     builder.env(containerEnvStrings());
     builder.exposedPorts(containerExposedPorts());
-    builder.volumes(volumes());
+    builder.volumes(volumes().keySet());
```

Comment:
why is this changed, wouldn't it lose the source part of the volume specification?

Reply:
The method accepting a `Map<String, Map>` [was deprecated](https://github.com/spotify/docker-client/blob/353417a693174296b4cba538e8c7ef5ae177b3e9/src/main/java/com/spotify/docker/client/messages/ContainerConfig.java#L278-L286).